src/components/frame/Extensions/layouts/partials/ColonySwitcherContent/ColonySwitcherContent.tsx

```diff
@@ -80,23 +41,37 @@ const ColonySwitcherContent: FC<ColonySwitcherContentProps> = ({ colony }) => {
           <ColonySwitcherItem
             name={name || ''}
             avatarProps={{
-              colonyImageProps: metadata?.avatar
-                ? { src: metadata?.thumbnail || metadata?.avatar }
+              colonyImageProps: colony?.metadata?.avatar
+                ? {
+                    src:
+                      colony?.metadata?.thumbnail || colony?.metadata?.avatar,
+                  }
                 : undefined,
-              chainIconName: chainIcon,
+              chainIconName,
             }}
             to={`/colony/${name}`}
           />
         </div>
       )}
       {joinedMoreColonies && (
         <div className="border-t border-t-gray-200 pt-6 flex flex-col gap-6">
-          {/* <div>add search here</div> */}
+          <SearchInput onInput={onInput} />
           <div>
             <h3 className={titleClassName}>
-              {formatText({ id: 'navigation.colonySwitcher.joinedColonys' })}
+              {formatText({ id: 'navigation.colonySwitcher.joinedColonies' })}
```

Comment:
Wouldn't it be better to create a key that is not dependent on translation content? Like `'navigation.colonySwitcher.heading` or something?